sys-block/sas3ircu/sas3ircu-16.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
Generally version bumps should move the ebuild to `EAPI=7`

Reply:
Fixed.